solutions/standard/provider.tf

```diff
@@ -1,6 +1,12 @@
 provider "ibm" {
-  alias            = "kms"
   ibmcloud_api_key = var.ibmcloud_api_key
   region           = var.region
   ibmcloud_timeout = 60
 }
+
+
+provider "ibm" {
+  alias            = "kms"
+  ibmcloud_api_key = var.ibmcloud_kms_api_key != null ? var.ibmcloud_kms_api_key : var.ibmcloud_api_key
+  region           = local.kms_region
+}
```

Comment:
This is inconsistent with the redis implementation.  ibmcloud_timeout and blank lines. https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs says the default is 60, so I think we can avoid coding it.

Reply:
I have removed timeout and extra blank line